go/client/logclient.go

```diff
@@ -271,10 +268,10 @@ func (c *LogClient) GetSTH() (sth *SignedTreeHead, err error) {
 	return
 }
 
-// Attempts to retrieve the entries in the sequence [|start|, |end|] from the CT
+// GetEntries attempts to retrieve the entries in the sequence [|start|, |end|] from the CT
```

Comment:
Apparently comment blocks 'typically' wrap at 80 for readability in the code, even though the code doesn't and godoc will sort it all out anyway.